src/buildtools/fsyacc/fsyaccast.fs

```diff
@@ -39,7 +39,7 @@ type Symbols = Symbol list
 //---------------------------------------------------------------------
 // Output Raw Parser Spec AST
 
-let StringOfSym sym = match sym with Terminal s -> "'" + s + "'" | NonTerminal s -> s
+let StringOfSym sym = match sym with Terminal s -> "'" ^ s ^ "'" | NonTerminal s -> s
```

Comment:
Is there a reason `^` is preferable?

Reply:
I don't think it is, it's just olf F# code

Reply:
This is copying over the canonical source in FsLexYacc repo - someone must have changed it in visualfsharp repo though they shouldn't have